### DISCUSSION TITLE
From Free Organization to Enterprise

### DISCUSSION THREAD

**Discussion by Fabman08** [community/community#129009] [Enterprise] [answered]
Hi all!
We're now approching to three free organizations in order to manage users, roles and Copilot seats. We would create 10 personal accounts and invite them to the organizations.
These accounts would have the company email address.
Our next step will be create an Enterprise account adding the three existing organization and enabling single sign on with Entra.
Can the existing personal accounts joins the accounts created via Entra by the email address?
Thank you!

**Comment by queenofcorgis** [comment, marked as answer]
Hello @Fabman08, 

Yes, provided you aren't creating an EMU Enterprise account. Once you create the Enterprise account, you'll transfer the orgs into it. From there your options are: 

* Enable SSO at the Enterprise level
* Enable SSO on the individual orgs 

Once SSO is enabled, the next time a user signs in with their personal account they should be redirected to Entra creating a linked identity. 

Some helpful docs: 
* [Inviting an Organization to join your Enterprise Account](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise#inviting-an-organization-to-join-your-enterprise-account)
* [Authentication with SAML SSO](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on#about-authentication-with-saml-sso)
* [Enabling and testing SAML SSO for your organization](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization)

**Reply by Fabman08** [reply]
Hello @queenofcorgis ,

thank you for the quick reply.
I hope to upgrade our environment to enterprise tier soon to enjoy its pros.

**Reply by Alva072** [reply]
> Hello @Fabman08,
> 
> Yes, provided you aren't creating an EMU Enterprise account. Once you create the Enterprise account, you'll transfer the orgs into it. From there your options are:
> 
> * Enable SSO at the Enterprise level
> * Enable SSO on the individual orgs
> 
> Once SSO is enabled, the next time a user signs in with their personal account they should be redirected to Entra creating a linked identity.
> 
> Some helpful docs:
> 
> * [Inviting an Organization to join your Enterprise Account](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise#inviting-an-organization-to-join-your-enterprise-account)
> * [Authentication with SAML SSO](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on#about-authentication-with-saml-sso)
> * [Enabling and testing SAML SSO for your organization](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization)

**Comment by Alva072** [comment]
> Hi all! We're now approching to three free organizations in order to manage users, roles and Copilot seats. We would create 10 personal accounts and invite them to the organizations. These accounts would have the company email address. Our next step will be create an Enterprise account adding the three existing organization and enabling single sign on with Entra. Can the existing personal accounts joins the accounts created via Entra by the email address? Thank you!

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
